packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -71,17 +71,24 @@
   }
 }
 
+/**
+ * @prop --atomic-crga-collapsed-height: The maximum height of the collapsed generated answer container.
+ */
 [part='generated-container'] {
   &.answer-collapsed {
-    @apply relative max-h-64 overflow-hidden content-[''];
+    @apply relative overflow-hidden content-[''];
+    max-height: var(--atomic-crga-collapsed-height, 16rem);
 
     .feedback-buttons {
       @apply hidden;
     }
   }
   &.answer-collapsed:before {
     @apply absolute left-0 top-0 h-full w-full content-[''];
-    background: linear-gradient(transparent 11.25rem, var(--atomic-background));
+    background: linear-gradient(
+      transparent calc(var(--atomic-crga-collapsed-height, 16rem) - 4.75rem),
```

Comment:
Instead of subtracting a fixed value (`4.75rem`), shouldn't we compute a ratio of the current maximum height (~70%)?
The main advantage I see, is that the gradient would still work when `--atomic-crga-collapsed-height` is smaller than `4.75rem`. Also, the gradient would be thinner as the component height decreases.

Reply:
Thats a good idea. Let me try that

Reply:
It works well! Updated the video in the description